### PR TITLE
Fix Active Storage update task when running in an engine

### PR DESCRIPTION
**Pull request by jmalcic**
Currently, running the update Rake task for Active Storage fails in Engines. This is because the task that needs to be called, `app:active_storage:install`, is namespaced. This is essentially like #31391 but for the update task.

### CODE REVIEW

**Comment by jmalcic** [comment]
Sorry deleted the branch by accident

**Comment by jmalcic** [comment]
As far as I can tell all the test failures in Buildkite are unrelated

File: activestorage/lib/tasks/activestorage.rake [hunk 1]
```diff
@@ -17,6 +17,10 @@ namespace :active_storage do
   task update: :environment do
     ENV["MIGRATIONS_PATH"] = "db/update_migrate"
 
-    Rake::Task["active_storage:install"].invoke
+    if Rake::Task.task_defined?("active_storage:install:migrations")
```
Comment (eugeneius) [inline activestorage/lib/tasks/activestorage.rake:20]:
```suggestion
    if Rake::Task.task_defined?("active_storage:install")
```

**Review by zzak — APPROVED** [review]

**Comment by eugeneius** [comment]
Would you mind adding a test for this? I guess it'll look almost identical to https://github.com/rails/rails/commit/c5b900da7323d33ae10e00c45ca7aaebc0e9c75c.

CI is broken because the branch is too far behind, a rebase will fix it. If you can, please squash your commits too.

**Comment by eugeneius** [comment]
Thank you @jmalcic! ✨

**Pull request merged**